src/qn/qnitensor.jl

```diff
@@ -32,6 +51,22 @@ ITensor(::Type{ElT}, inds::QNIndex...) where {ElT<:Number} =
 
 ITensor(inds::QNIndex...) = ITensor(Float64, QN(), IndexSet(inds...))
 
+"""
+    ITensor([::Type{ElT} = Float64, ]::UndefInitializer, inds)
```

Comment:
Looks like you forgot the `flux::QN` input here.

Reply:
Good catch! Just fixed this in another PR today.